### ISSUE TITLE
"Cannot find twitter account with name = ",  This happened suddenly today, but it was fine yesterday.

### ISSUE THREAD

**Issue by Lxhnnn** [twintproject/twint#863]
# Issue Template
Please use this template!

### Initial Check
> If the issue is a request please specify that it is a request in the title (Example: [REQUEST] more features). If this is a question regarding 'twint' please specify that it's a question in the title (Example: [QUESTION] What is x?). Please **only** submit issues related to 'twint'. Thanks.

>Make sure you've checked the following:

- [] Python version is 3.6;
- [] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
>Please provide the _exact_ command ran including the username/search/code so I may reproduce the issue.

### Description of Issue
>Please use **as much detail as possible.**

### Environment Details
>Using Windows, Linux? What OS version? Running this in Anaconda? Jupyter Notebook? Terminal?

**Comment by THEDARKK** [comment]
Same

**Comment by Legnus** [comment]
Same here

**Comment by andywalden** [comment]
This issue is due to Twitter disabling the legacy interface and redirecting to the javascript page. At a minimum, anything scraping twitter without an API now must at least click the "Yes" button on the direct page. This change has broken this app entirely and made it obsolete.

**Comment by mikaelyh** [comment]
Even though the disabling of the legacy interface, it was working until yesterday.
Something new has broken this app.

**Comment by javad94** [comment]
> Even though the disabling of the legacy interface, it was working until yesterday.
> Something new has broken this app.

Maybe because they launched their new API.

**Comment by skywd** [comment]
> > Even though the disabling of the legacy interface, it was working until yesterday.
> > Something new has broken this app.
> 
> Maybe because they launched their new API.

Seems as though they have completely phased out requesting that page with XML. Request returns 400 error for me now.

**Comment by andywalden** [comment]
Yes, they disabled the ability to access the "legacy site" without a javascript "yes", clickthough. And then even the legacy site is sorely lacking granularity. You don't get a real timestamp, the pics, videos and other media aren't parsed out and threads are presented differently. It's a very primitive set of data to work from,

I don't actually use twint as I found it to be too restrictive for what I was trying to do, but obviously I'm watching the team to see how they react to this change. I maintain my own twitter bot and have converted the mobile.twitter.com experience to my objects, albeit, with reduced information.

**Comment by neuraleye** [comment]
Has the ability to crawl likes without authentication been completely nuked?

**Comment by andywalden** [comment]
It does not appear that likes, nor any stateful info, is available on the minimized mobile interface. 

Puppeteer may be the next frontier:
 https://github.com/puppeteer/puppeteer#faq

**Comment by neuraleye** [comment]
Thanks, and does twitter lock accounts for suspicious behavior like scrolling 1K people's likes in an hour?

**Comment by medhax** [comment]
Same issue here, in my case is also giving CRITICAL....: 'NoneType' object is not suscriptable.

Hope to get it restored soon :)

**Comment by rossrochford** [comment]
@andywalden I'm writing currently a Twitter scraper using Selenium with a custom asyncio wrapper. It's working well but getting a users' followers requires logging in. Happy to open source my progress and collaborate.

**Comment by bo-tadpol** [comment]
@pielco11 I know you have mentioned in the past you have very little free time right now, do you think you have some bandwidth to help breath some life back into twint?

**Comment by yunusemrecatalcam** [comment]
I see that twitter legacy theme is not closed yet for favorites/followings/search pages. This means we only lost the profile page for now. Since other functions need user id in order to fetch the data, we still need to able to fetch profile page. I am trying to figure out a way to fetch profile page data without login or using webdriver.
For those who interested, when you open a profile from browser;
It uses a guest token named 'x-guest-token'  to query an endpoint ('https://api.twitter.com/graphql/-xfUfZsnR_zqjFd-IfrN5A/UserByScreenName?variables=%7B%22screen_name%22%3A%22LeagueOfLegends%22%2C%22withHighlightedLabel%22%3Atrue%7D') that returns user data as json.
firstly I have checked the timeout for that token, unluckily its died in a few hours. Now I am trying to get the token before every profile request.   
Anyone working on that, I'd be really happy to talk about that and collaborate.
Here is a discord link if you wanna closely communicate and rescue the twint together! We'll hack it out again!
https://discord.gg/ze4Zjj

**Comment by darvell** [comment]
@yunusemrecatalcam Your discord link is expired but this shouldn't be much of a project. I haven't documented this properly so it's vague but a config endpoint is hit even if you're not logged in, you get an auth token among other things so you're "logged in" and without that, things break.

**Comment by pielco11** [comment]
`twint -s "from:username"` replaces `twint -u username`, works as expected

**Comment by THEDARKK** [comment]
But to make the --following command work, it's no longer possible.

sudo twint -u username --following -o test.txt

**Comment by pielco11** [comment]
You are right. But consider that the `--following/--followers` is intended to be more a PoC rather than else, honestly idk why people don't use the official API which allows you to get 15k followers/following in a couple of seconds


Usually I don't suggest to run twint as sudo, better install in the user's PATH

**Comment by subokita** [comment]
Similar issue here, also using "from:username" doesn't work on all users, even using Twitter's own search field.

**Comment by darvell** [comment]
Maybe we just need to modify twint to require user/pass just for the tokens?

**Comment by pielco11** [comment]
@subokita the `from` parameter is the one used by Twitter, so it's expected to work. In my test it works well so idk what you are getting

**Comment by subokita** [comment]
@pielco11 Yes, I used that "from" params for search very regularly for the past year, but I found out that, at least on my case, it doesn't work 100%, it won't surface any tweets from some valid active users, e.g. for my case "from:yellowkenworth" returns nothing, while yellowkenworth is an active user, not suspended or anything.

**Comment by pielco11** [comment]
I guess that your target has been shadow-banned, so his/her tweets do not compare in search results

![image](https://user-images.githubusercontent.com/17363750/90417187-b47bc700-e0b3-11ea-9197-866e649359ab.png)

**Comment by DiabetesLongTorsoMan** [comment]
@pielco11 
I'm new to this library and you seem to know what's going on. Is this kind of issue typical where it causes much of the library to no longer work? And the issue isn't resolved for a while? I'm basically just wondering how long it usually takes for stuff like this to be resolved, and if it's likely that this library will stay broken due to the changes twitter made. 
Thank you!!!

**Comment by darvell** [comment]
I wonder if a logged in user would work.

**Comment by yunusemrecatalcam** [comment]
> You are right. But consider that the `--following/--followers` is intended to be more a PoC rather than else, honestly idk why people don't use the official API which allows you to get 15k followers/following in a couple of seconds
> 
> Usually I don't suggest to run twint as sudo, better install in the user's PATH

@pielco11 twitter api is limited when you want to scrape users on a scale.
I have created a pull request that makes it possible to fetch favorite and followings again, lets check it out!

**Comment by shadow-samurai** [comment]
please help --following/--followers command in twitter don`t run and show error ValueError: Cannot find twitter account with name =XXXX 
I checked with 10 IP and don`t run 
twitter api limit

**Comment by pielco11** [comment]
@yunusemrecatalcam 
Merged

@DiabetesLongTorsoMan 
tweets of shadowbanned users are not found not due to twint fail

**Comment by somepasta** [comment]
After updating twint from the latest commit I tried getting a list of followers/following using the `--followers/--following` commands. However, the error CRITICAL:root:twint.feed:Follow:IndexError shows up every single time. The full command I'm trying to use is `twint -s "from:username" --following --user-full -o file.txt`

**Comment by yunusemrecatalcam** [comment]
Follower page is not stable for now. I'm gonna check it

**Comment by pielco11** [comment]
@somepasta the `-s` argument is used just for the searches, for that trick does not work for followers/following

**Comment by somepasta** [comment]
@yunusemrecatalcam Thank you.

@pielco11 I see. I thought replacing with `-s "from:username"` could possibly solve this problem. Using `-u username` will always return in `ValueError: Cannot find twitter account with name = username` as mentioned above. Hoping there's a new way to scrape a list followings/followers soon, it was extremely useful and simple to use.

**Comment by shadow-samurai** [comment]
i want use from twint for get followers/following please help me
twint show error : CRITICAL:root:twint.feed:Follow:IndexError

**Comment by pielco11** [comment]
Lookup function works partially (some fields can't be scraped)
```python
import twint

c = twint.Config()
c.Username = 'NaturalIntelli4'
c.Store_object = True
c.Store_object_users_list = []
c.User_full = True
twint.run.Lookup(c)
user_info = c.Store_object_users_list[0]
```

Followers/following working if `c.User_full = False` and the user ID is specified

```python
c = twint.Config()
c.User_id = "2550600458"
c.Username = "noneprivacy"
c.Limit = 20
twint.run.Following(c)
```

Please update with the pip+git
Thanks to @yunusemrecatalcam for fixing

**Comment by shadow-samurai** [comment]
--user-full was very good switch in twint

**Comment by glazou** [comment]
To fetch tweet of a given user, the configuration that works for me is:
```
twint_config = twint.Config()
twint_config.Search = f"from:{username}"
twint.run.Search(twint_config)
```

**Comment by alexgeo20** [comment]
> `twint -s "from:username"` replaces `twint -u username`, works as expecte


Thank you very much for this!  This worked for me.

**Comment by alexgeo20** [comment]
@pielco11 thank you very much for your help with this.  I tried it with `twint -s "from: username"` as you mentioned and it worked!

**Comment by mikaelyh** [comment]
violation of antitrust laws

https://www.reuters.com/article/us-microsoft-linkedin-ruling-idUSKCN1AU2BV

**Comment by GregCottingham** [comment]
Thank you, `twint -s "from: username"` does work well for looking at a specific username. Is there a way to further refine specific search terms for a targeted username? Something like:  `twint -s "from: username" -s searchterm` ?

**Comment by coltranetorres** [comment]
just encountered "Cannot find twitter account with name =" again just today, i've already upgraded to the latest version of twint, Search() does not work but Lookup() does work, please help me resolve this:(

**Comment by coltranetorres** [comment]
> just encountered "Cannot find twitter account with name =" again just today, i've already upgraded to the latest version of twint, Search() does not work but Lookup() does work, please help me resolve this:(

i tried both in jupyter notebook and command line already but still does not work:((
